airbyte-integrations/connectors/source-mysql-v2/src/test/resources/expected-spec.json

```diff
@@ -0,0 +1,293 @@
+{
+  "documentationUrl": "https://docs.airbyte.com/integrations/sources/mysql",
+  "connectionSpecification": {
+    "type": "object",
+    "title": "Mysql Source Spec",
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "required": [
+      "host",
+      "port",
+      "username",
+      "encryption",
+      "tunnel_method",
+      "cursor"
+    ],
+    "properties": {
+      "host": {
+        "type": "string",
+        "order": 1,
+        "title": "Host",
+        "description": "Hostname of the database."
+      },
+      "port": {
+        "type": "integer",
+        "order": 2,
+        "title": "Port",
+        "default": 3306,
+        "maximum": 65536,
+        "minimum": 0,
+        "description": "Port of the database.\nMysql Corporations recommends the following port numbers:\n1521 - Default listening port for client connections to the listener. \n2484 - Recommended and officially registered listening port for client connections to the listener using TCP/IP with SSL."
```

Comment:
mysql corporations?

Reply:
lol a simple mass replace caused this. I'll remove the recommendation part.